SPECS/fwctl-hwe/fwctl-hwe.spec

```diff
@@ -88,9 +88,8 @@ BuildRequires:  binutils
 BuildRequires:  systemd
 BuildRequires:  kmod
 BuildRequires:  mlnx-ofa_kernel-hwe-devel = %{_mofed_full_version}
-BuildRequires:  mlnx-ofa_kernel-hwe-source = %{_mofed_full_version}
 
-Requires:       mlnx-ofa_kernel-hwe = %{_mofed_full_version}
+Requires:       mlnx-ofa_kernel
```

Comment:
it seems you didn't change this line in fwctl.spec and all the rest of the default kernel OOT module builds. Is that intentional?

Reply:
Good eye, yes, it was intentional. I wanted to limit the changes as much as possible to not introduce regressions in packages, that are already published and working. And I'd love that clean-up to happen across other kmod specs, just not in this PR.